src/include/miopen/solver/implicitgemm_ck_util.hpp

```diff
@@ -605,15 +637,15 @@ ConvSolution InitInvokerFactoryNCHW(const ExecutionContext& ctx,
                 std::swap(conv_tensors.x, conv_tensors.y);
                 std::swap(conv_tensors.xDesc, conv_tensors.yDesc);
             }
-
+#ifdef WORKAROUND_CK_ISSUE_1184
+            HipEventProfiler pfr(handle);
+#endif
             input1_tr_inst.ConvertFrom(handle, kernels, conv_tensors);
 
             input2_tr_inst.ConvertFrom(handle, kernels, conv_tensors);
 
             output_init_tr_inst.ConvertFrom(handle, kernels, conv_tensors);
 
-            /// \todo: Fix NHWC Wrw invokers to also issue a zero-out kernel. Will
-            /// need SetTensor() to properly zero out non-packed tensors
```

Comment:
Please keep the second statement of this TODO. It is about replacing ZeroOutBuffer with SetSensor here